blue_modules/notifications.ts

```
@@ -0,0 +1,629 @@
+};
+
+/**
+ * Retrieves the notification levels from GroundControl.
+ *
+ * @returns {Promise<Record<string, any>>} The levels object
+ */
+const getLevels = async (): Promise<Record<string, any>> => {
+  const pushToken = await getPushToken();
+  if (!pushToken || !pushToken.token || !pushToken.os) return {};
+
+  let response: void | Response;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)